src/popup/bookmarkModule.js

```diff
@@ -262,11 +262,12 @@ function loadImages() {
             checkbox.isChecked = !checkbox.isChecked; // Update isChecked Property in checkbox
 
             // Update SelectAll Button
-            if (selectedBookmarks === elements.selectCheckboxes.length) {
-              elements.buttonSelectAllCheckbox[0].children[0].innerText = 'Deselect All';
-            } else {
+            if (selectedBookmarks !== elements.selectCheckboxes.length) {
```

Comment:
This if statement is wrong. The only reason it does not break anything is that it is written above the `if (selectedBookmarks > 0)` statement. After swapping these two, you can see the problem in the working. The condition inside this if statement should be `(selecteBookmarks === 0)`.